using_images/xpaas_images/sso.adoc

```diff
@@ -0,0 +1,166 @@
+= Red Hat SSO xPaaS Image
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+[IMPORTANT]
+This image is currently in https://access.redhat.com/support/offerings/techpreview[Technical Preview] and not intended for production use.
+
+== Overview
+
+Red Hat Single Sign-On (SSO) is an integrated sign-on solution available as a containerized xPaaS image designed for use with OpenShift. This image provides an authentication server for users to centrally login, logout, register, and manage user accounts for web applications, mobile applications, and RESTful web services.
+
+Red Hat offers five SSO Application Templates, each of which deploys two SSO web server pods and a separate pod for the backing database:
+
+* *_sso70-basic_* - SSO backed by a H2 database
+* *_sso70-mysql_* - SSO backed by a MySQL database
+* *_sso70-mysql-persistent_* - SSO backed by a persistent PostgreSQL database
+* *_sso70-postgresql_* - SSO backed by a MySQL database
+* *_sso70-postgresql-persistent_* - SSO backed by a persistent PostgreSQL database
+
+An SSO-enabled Red Hat JBoss Enterprise Application Platform (JBoss EAP) Image has been offered which enables users to deploy a JBoss EAP instace that can use SSO for authentication:
+
+* *_eap64-sso-s2i_* - SSO-enabled JBoss EAP
+
+== Differences Between the OpenShift Red Hat Single Sign-On xPaaS Application and Keycloak
+The OpenShift Red Hat Single Sign-On xPaaS application is built from Keycloak, a JBoss community project. There are some differences in functionalitity between the Red Hat Single Sign-On xPaaS Application and Keycloak:
+
+* This image is currently available as a Technical Preview for use only with SSO-enabled Red Hat JBoss Enterprise Application Platform (JBoss EAP) applications.
```

Comment:
Leaving as caps to attract attention and because it fits consistency with the Tech Preview link from the earliest admonition